ceph-pull-requests-arm64/config/definitions/ceph-pull-requests-arm64.yml

```diff
@@ -0,0 +1,100 @@
+- job:
+    block-downstream: false
+    block-upstream: false
+    builders:
+    - shell: |-
+        bash -ex
+        NPROC=$(nproc)
+        testnproc=$(($NPROC / 4))
+        #export CHECK_MAKEOPTS="-R unittest* -j${testnproc}"
+        export CHECK_MAKEOPTS="-j${testnproc}"
+        export BUILD_MAKEOPTS="-j${NPROC}"
+        ./run-make-check.sh
+        sleep 5
```

Comment:
we usually don't have shell directives that are longer than 1-2 lines. When that is the case we create a script in <job name>/build/build and then pull it in. Makes it easier to read too. The way you do that is like:

      - shell:
          !include-raw:
            - ../..//build/build